modules/aws/ami.go

```diff
@@ -63,28 +66,34 @@ type imageSort []*ec2.Image
 func (a imageSort) Len() int      { return len(a) }
 func (a imageSort) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
 func (a imageSort) Less(i, j int) bool {
-	itime, _ := time.Parse(time.RFC3339, *a[i].CreationDate)
-	jtime, _ := time.Parse(time.RFC3339, *a[j].CreationDate)
-	return itime.Unix() < jtime.Unix()
+	iTime, err := time.Parse(time.RFC3339, *a[i].CreationDate)
+	if err != nil {
+		panic(fmt.Errorf("error parsing creation date of EC2 image: %s", err))
+	}
+	jTime, err := time.Parse(time.RFC3339, *a[j].CreationDate)
+	if err != nil {
+		panic(fmt.Errorf("error parsing creation date of EC2 image: %s", err))
```

Comment:
We do not want to `panic` in Terratest, ever, if we can avoid it. It will prevent cleanup code from running, such as the call to `terratest.Destroy`, and therefore, leave lots of resources running in your AWS account (or wherever you're testing). 

If we're going to make this change, then anything that uses `imageSort` needs to `recover` from possible panics, and we should probably have an automated test to verify it actually works.

Reply:
Good point. So I guess, in this case, it's preferable just not to handle the error...

Reply:
It's crappy either way, but yea, probably.

Reply:
Ok. I removed it.